src/engine.py

```
@@ -9172,31 +9172,49 @@ async def run_turn(transcript_text: str) -> None:
+                            try:
+                                override_cfg = (llm_options or {}).get("hangup_call_guardrail_markers")
+                                override_end = None
+                                if isinstance(override_cfg, dict):
+                                    override_end = override_cfg.get("end_call")
+                                else:
+                                    override_end = override_cfg
+                                if override_end:
+                                    end_markers = normalize_marker_list(override_end, list(end_markers))
+                                    end_markers_source = "pipeline_override"
+                            except Exception:
+                                pass
```

Ruff (0.15.0)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)